test/mpi/Makefile.am

```diff
@@ -19,9 +19,9 @@ all_lang_subdirs = f77 cxx f90 f08
 # duplication because automake is able to "see" into simple variable
 # assignments that are not driven by a configure @-substitution variable.
 DIST_SUBDIRS = $(static_subdirs) io $(all_lang_subdirs) threads spawn .
-SUBDIRS      = $(static_subdirs) $(iodir) $(otherlangs) $(threadsdir) $(spawndir) $(ckpointdir) $(ftdir) .
+SUBDIRS      = $(static_subdirs) $(iodir) $(otherlangs) $(threadsdir) $(spawndir) $(ckpointdir) ft .
```

Comment:
that was intended.

if ```SUBDIRS``` does not contain ```ft```, and if the test suite is configure'd with ```--disable-ft-tests```, then ```make dist``` will fail because of the missing ```ft``` directory.
so i think the right approach is to always include the ```ft``` directory, and __not__ build the ft related tests if they are disabled.
see the full commit at ggouaillardet/mpich@441853771c322278b9c819c001eeb2998cc53aa7

and i should have applied the same logic to all the directories.
if that makes sense, i will do so and update the PR